lib/runtime-core/src/codegen.rs

```diff
@@ -296,6 +364,7 @@ impl MiddlewareChain {
         fcg: Option<&mut FCG>,
         ev: Event,
         module_info: &ModuleInfo,
+        loc: u32,
```

Comment:
Are we changing the public API? If so, is there any way to either:
* Have this added into the CHANGELOG as [BREAKING CHANGE]
* Minimize this? (not sure how, we will have to research)